engine/src/main/java/org/terasology/engine/world/block/family/AbstractBlockFamily.java

```diff
@@ -16,12 +16,7 @@ public abstract class AbstractBlockFamily implements BlockFamily {
     private BlockUri uri;
     private Set<String> categories = Sets.newHashSet();
 
-    protected AbstractBlockFamily(BlockFamilyDefinition definition, BlockShape shape, BlockBuilderHelper blockBuilder) {
-        setBlockUri(new BlockUri(definition.getUrn()));
-        setCategory(definition.getCategories());
-    }
-
-    protected AbstractBlockFamily(BlockFamilyDefinition definition, BlockBuilderHelper blockBuilder) {
+    protected AbstractBlockFamily(BlockFamilyDefinition definition) {
```

Comment:
change unrelated to the commit message - is that on purpose?

Reply:
yes and no - I saw that `blockBuilder` isn't used and wanted to change something about it but then I thought I'd ask soundwave about the intent of the changes first - still waiting to hear back... wanted to put the change in a separate PR, but seems they snuck their way in here unfortunately...

Reply:
reverted